plutus-playground-client/test/CursorTests.purs

```diff
@@ -41,13 +40,6 @@ applyOperation Right = Cursor.right
 applyOperation First = Cursor.first
 applyOperation Last = Cursor.last
 
-genIndex :: forall m a. MonadGen m => Cursor a -> m Int
-genIndex cursor = chooseInt 0 (Cursor.length cursor - 1)
-
--- | Generate an index which is roughly in range, but may be slightly out of bounds.
-genLooseIndex :: forall m a. MonadGen m => Cursor a -> m Int
-genLooseIndex cursor = chooseInt (-5) (Cursor.length cursor + 5)
-
```

Comment:
why are these being removed?

Reply:
They've been generalised and moved to `TestUtils`. They now work for any `Foldable`. (Motivated by the need for the exact same function for `Array`.)